Yona/Yona/FriendsProfileMasterView.swift

```diff
@@ -63,19 +63,20 @@ extension FriendsProfileMasterView {
         }
         else {
             //TODO: return profile notifications counts
-            return self.buddiesOverviewArray.count
+            return 1
         }
         
     }
     
     func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
         let cell: UITableViewCell = tableView.dequeueReusableCellWithIdentifier("cell", forIndexPath: indexPath)
+        if isFromFriends == true {
```

Comment:
I fixed this one in the same way...so whoever merges this will need to resolve the conflict with the main branch...unless you can revert this Chandan